app/helpers/application_helper.rb

```diff
@@ -12,26 +12,28 @@ def view_class
   end
 
   def core_pages?
```

Comment:
I think the method's name should be singular.
Refer to [this comment](https://github.com/thepracticaldev/dev.to/pull/332#issuecomment-412296031)

Reply:
I too, but did not change

Reply:
@rhymes, what do you have to say on this?

Reply:
@ocnly see too https://github.com/thepracticaldev/dev.to/pull/332